pdf_build_src/process_markdowns.py

```diff
@@ -679,33 +681,38 @@ def process_macros(duplicated_src_dir_path):
 
     duplicated_src_dir_path = "src_copy/src"
 
-    # Step 1: make a copy of the src directory in the current directory
+    # make a copy of the src directory in the current directory
```

Comment:
Removed the step ordering... Because we will have to change it everytime steps are added.